components/items/ItemsGrid/ItemsGrid.vue

```diff
@@ -23,7 +23,8 @@
           :variant="
             (slotProps.isMobileVariant || slotProps.grid === 'small') &&
             'minimal'
-          " />
+          "
+          :unlockable="index < 5" />
```

Comment:
Dont understand this sir

Reply:
draft.
there is no data source for unlockable.
@yangwao  did i misunderstand the requirements?

Reply:
This is, afaik, what was required; otherwise, I would be unable to review.